### PR TITLE
 #158456314 Implement Travis Continuous Integration

### PR DESCRIPTION
**Pull request by RrNn**
### What does this PR do?
- Integrate Continuous integration using Travis and test coverage using Coveralls
- Show the badges in the readme, for Travis CI and Coveralls
### Relevant pivotaltracker story
[#157273805](https://www.pivotaltracker.com/n/projects/158456314)

### CODE REVIEW

**Review by solnsubuga — APPROVED** [review]
LGTM

**Review by wasswa-derick — COMMENTED** [review]
- 'hc-venv' in the gitignore, you can consider opening your project at the root of hc-wits-kla, that way, the virtual environment folder won't even be tracked.
- Or you can consider using virtualenvwrapper. It allows to create various virtual environments and switch between them anywhere, hence there will be no need to track it in gitignore.

**Review by wasswa-derick — APPROVED** [review]

**Comment by ghost** [comment]
Good work

**Review by Yiga-fred — APPROVED** [review]

**Review by Yiga-fred — APPROVED** [review]
Good work

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,6 @@ hc/local_settings.py
 static-collected
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+hc-venv
+.vscode/
```
Comment (john555) [inline .gitignore]: Add a new line at the end of this file.

**Review by john555 — CHANGES_REQUESTED** [review]
Overall, PR looks good.

File: .gitignore (hunk 1 above)
Comment (RrNn) [reply .gitignore]: Okay, I'm gonna do that

**Comment by john555** [comment]
Resolve conflicts on your PR.

**Review by john555 — APPROVED** [review]

**Pull request merged**